CIP-Liesenfelt-Shelleys_Voltaire_decentralization_update/README.md

```diff
@@ -260,7 +260,84 @@ The winners of block forks and slot battles are determined by which pool has a l
 
 ## Leverage-based Pool Ranking Recommendation
 
-Pool ranking scores in wallet browsers have a significant impact on delegator choices. When pledge becomes the most important factor for total pool size lower leverage factors are more desirable. Lists should be sorted by leverage and presented in an ascending order with the lowest leverage pools first.
+Pool ranking scores in wallet browsers have a significant and powerful game theory impact on delegator pool choices, yet it is often overlooked and not transparent in wallets.
+
+When pledge becomes the most important factor for total pool size, lower leverage factors are more desirable. Lists should be sorted by leverage and presented in an ascending order with the lowest leverage pools first.
+
+The following is guidance for what a pool ranking system should be prioritizing pledge and low leverage.
+
+### Ranking Equation
+
+The recommended ranking equation starts with a highest score of 10. The pools are down-ranked solely based on leverage and fees factors.
+
+    //equation
+    ranking_score = 10 - leverage_factor_term - fee_factor_term
+
+    //variables
+    leverage_factor_term = 10 * max{ (pool_leverage/L)^A, (pool_stake/saturation_stake)^B }
+    fee_factor_term = C * pool_fee_margin
+
+    //parameters
+    - A is 2, has range (0,10), can be tweaked
+    - B is 2, has range (0,10), can be tweaked
```

Comment:
A and B likely will be floats. Anytime a float is desired use 2.0 instead of 2. Range should be ( 0.0, 10.0 ) otherwise the assumption is integers.

Reply:
good catch, updated.